safe/gui/tools/multi_buffer_dialog.py

```diff
@@ -0,0 +1,235 @@
+# coding=utf-8
+"""**Volcano Buffer Implementation.**
+
+"""
```

Comment:
Landscape will make you warning for this one : 
```"""Volcano Buffer Implementation."""```
It needs to be on the same line.